js/image/shared/ImageUploader.svelte

```diff
@@ -59,6 +60,13 @@
 		img_blob: Blob | any,
 		event: "change" | "stream" | "upload"
 	): Promise<void> {
+		if (event === "stream") {
+			dispatch("stream", {
+				value: { url: img_blob } as Base64File,
```

Comment:
So if its a stream event, it will always upload the value as a base64 string? I thought it was only if the component also sets `format="base64"`?

Reply:
Oh I see, so the `format="base64` is only necessary if you are streaming _out_ an image